elide-standalone/src/main/java/com/yahoo/elide/standalone/ElideStandalone.java

```diff
@@ -121,7 +125,8 @@ public void start(boolean block) throws Exception {
                     "/*",
                     EnumSet.of(DispatcherType.REQUEST));
 
-            context.setAttribute(HealthCheckServlet.HEALTH_CHECK_REGISTRY, ElideResourceConfig.getHealthCheckRegistry());
+            context.setAttribute(HealthCheckServlet.HEALTH_CHECK_REGISTRY,
+                    ElideResourceConfig.getHealthCheckRegistry());
```

Comment:
For example here is a double tab.

Reply:
I think that is intellij doing that for line continuations. We should fix all the indents in the project.